src/bo4e/bo/ausschreibung.py

```diff
@@ -0,0 +1,99 @@
+    ausschreibungsstatus: Ausschreibungsstatus = attr.ib(validator=attr.validators.instance_of(Ausschreibungsstatus))
+
+    #: Kennzeichen, ob die Ausschreibung kostenpflichtig ist
+    kostenpflichtig: bool = attr.ib(validator=attr.validators.instance_of(bool))
+    #: Gibt den Veröffentlichungszeitpunkt der Ausschreibung an
+    veroeffentlichungszeitpunkt: datetime = attr.ib(validator=attr.validators.instance_of(datetime))
+
+    ausschreibender: Geschaeftspartner = attr.ib(validator=attr.validators.instance_of(Geschaeftspartner))
+    """
+    Mit diesem Objekt können Geschäftspartner übertragen werden.
+    Sowohl Unternehmen, als auch Privatpersonen können Geschäftspartner sein
+    """
+
+    abgabefrist: Zeitraum = attr.ib(validator=attr.validators.instance_of(Zeitraum))
+    """
+    Diese Komponente wird zur Abbildung von Zeiträumen in Form von Dauern oder der Angabe von Start und Ende verwendet.
+    Es muss daher entweder eine Dauer oder ein Zeitraum in Form von Start und Ende angegeben sein
+    """
+
+    bindefrist: Zeitraum = attr.ib(validator=attr.validators.instance_of(Zeitraum))
+    """
+    Diese Komponente wird zur Abbildung von Zeiträumen in Form von Dauern oder der Angabe von Start und Ende verwendet.
+    Es muss daher entweder eine Dauer oder ein Zeitraum in Form von Start und Ende angegeben sein
+    """
+
+    #: Die einzelnen Lose, aus denen sich die Ausschreibung zusammensetzt
+    lose: List[Ausschreibungslos] = attr.ib(
+        validator=attr.validators.deep_iterable(
+            member_validator=attr.validators.instance_of(Ausschreibungslos),
+            iterable_validator=check_list_length_at_least_one,
+        )
+    )
+    # optional attributes
+    #: Aufzählung der unterstützten Ausschreibungsportale
+    ausschreibungportal: Optional[Ausschreibungsportal] = attr.ib(
+        default=None, validator=attr.validators.optional(attr.validators.instance_of(Ausschreibungsportal))
+    )
+    #: Internetseite, auf der die Ausschreibung veröffentlicht wurde (falls vorhanden)
+    webseite: Optional[str] = attr.ib(
+        default=None, validator=attr.validators.optional(attr.validators.instance_of(str))
+    )
+
+
+class AusschreibungSchema(GeschaeftsobjektSchema):
+    """
+    Schema for de-/serialization of Ausschreibung
+    """
+
+    class_name = Ausschreibung
+
+    # required attributes
+    ausschreibungsnummer = fields.Str()
+    ausschreibungstyp = EnumField(Ausschreibungstyp)
+    ausschreibungsstatus = EnumField(Ausschreibungsstatus)
+    kostenpflichtig = fields.Bool()
+    veroeffentlichungszeitpunkt = fields.DateTime()
+    ausschreibender = fields.Nested(GeschaeftspartnerSchema)
+    abgabefrist = fields.Nested(ZeitraumSchema)
+    bindefrist = fields.Nested(ZeitraumSchema)
+    lose = fields.List(fields.Nested(AusschreibungslosSchema))
+
+    # optional attributes
+    ausschreibungportal = EnumField(Ausschreibungsportal, load_default=None)
+    webseite = fields.Str(load_default=None)
```

Comment:
The `load_default` approach will be deprecated in newer marshmallow versions: https://marshmallow.readthedocs.io/en/stable/upgrading.html?highlight=allow_none#upgrading-to-2-0
```suggestion
    ausschreibungportal = EnumField(Ausschreibungsportal, allow_none=True)
    webseite = fields.Str(allow_none=True)
```